fe/fe-core/src/main/java/org/apache/doris/datasource/InternalCatalog.java

```diff
@@ -448,8 +447,8 @@ public void createDb(String dbName, boolean ifNotExists, Map<String, String> pro
                 }
                 try {
                     unprotectCreateDb(db);
-                    CreateDbInfo dbInfo = new CreateDbInfo(InternalCatalog.INTERNAL_CATALOG_NAME, db.getName(), db);
-                    Env.getCurrentEnv().getEditLog().logCreateDb(dbInfo);
+                    // CreateDbInfo dbInfo = new CreateDbInfo(InternalCatalog.INTERNAL_CATALOG_NAME, db.getName(), db);
```

Comment:
remove this line directly?

Reply:
I prefer to keep it, in case I forget why i change this.